client/components/product-image/index.js

```diff
@@ -0,0 +1,50 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const ProductImage = ( { product, alt, width, height, className, ...props } ) => {
+	// The first returned image from the API is the featured/product image.
+	const productImage = product && product.images && product.images[ 0 ];
+	const src = ( productImage && productImage.src ) || false;
+
+	const classes = classnames( 'woocommerce-product-image', className, {
+		'is-placeholder': ! src,
+	} );
+
+	return (
+		<img
+			className={ classes }
+			src={ src || wcSettings.wcAssetUrl + 'images/placeholder.png' }
+			width={ width }
+			height={ height }
+			alt={ alt }
```

Comment:
It looks like [the product images object](https://woocommerce.github.io/woocommerce-rest-api-docs/#product-images-properties) should also include an `alt`, can we fall back to this if none is provided as a prop? (we might want to otherwise override it, like forcing it to `alt=""` if we know it's a decorative image)